gitversion.yml

```diff
@@ -1,3 +1,5 @@
+next-version: 0.0.0
```

Comment:
I didn't need this in the .NET project, so it's probably not necessary; possibly causing the problem, not sure.

Reply:
I was just reading through that related issue with `gitversion`.  It looks like pre-1.0.0 is a special case.  I think we’ll just have to do a manual tag for the first one.